### PR TITLE
Polish up brussels metadata

### PR DESCRIPTION
**Pull request by JosseVanDelm**
Solve following issues for brussels metadata (nl & fr):
* #22 
* #23 
* #24 
* #25

### CODE REVIEW

File: openaddresses/be/vlg/bosa-be-fr.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+    "data": "FIXME insert correct url here",
```
Comment (barthanssens) [inline openaddresses/be/vlg/bosa-be-fr.json]: Also include the URL here (https://opendata.bosa.be/download/best/openaddress-bebru.zip)

File: openaddresses/be/bru/bosa-be-fr.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{
```
Comment (barthanssens) [inline openaddresses/be/bru/bosa-be-fr.json]: Suggestion to change the file name to bosa-region-brussels-fr.json etc to be more in line with openaddresses guidelines on their wiki (even if they don't seem to enforce it in the github...)

**Review by barthanssens — CHANGES_REQUESTED** [review]

File: openaddresses/be/vlg/bosa-be-fr.json (hunk 1 above)
Comment (JosseVanDelm) [reply openaddresses/be/vlg/bosa-be-fr.json]: These files are for the flemish region, so i made the links:
* BE-VLG -->  https://opendata.bosa.be/download/best/openaddress-bevlg.zip
* BE-WAL --> https://opendata.bosa.be/download/best/openaddress-bewal.zip

File: openaddresses/be/bru/bosa-be-fr.json (hunk 2 above)
Comment (JosseVanDelm) [reply openaddresses/be/bru/bosa-be-fr.json]: Will rename these!

**Review by barthanssens — APPROVED** [review]

**Pull request merged**